src/Illuminate/Auth/Passwords/PasswordBroker.php

```diff
@@ -42,9 +42,10 @@ public function __construct(TokenRepositoryInterface $tokens, UserProvider $user
      * Send a password reset link to a user.
      *
      * @param  array  $credentials
+     * @param  \Closure|null  $callback
```

Comment:
Can this be `callable|null`?

Reply:
I just wanted to keep it in line with the other interface method.